tools/docker/generate_dockerfiles.py

```diff
@@ -886,15 +887,27 @@ def install_deps_spack(version: str) -> str:
 ENV PATH="/opt/venv/bin:$PATH"
 RUN pip3 install --quiet boto3==1.38.11 google-cloud-storage==3.1.0
 
-# Install a recent Spack version
+# Install Spack and Spack packages
 WORKDIR /root/spack
 ARG SPACK_VERSION
-ENV SPACK_VERSION=${{SPACK_VERSION:-develop-2025-05-18}}
-RUN git init --quiet && \
-    git remote add origin https://github.com/spack/spack.git && \
-    git fetch --quiet --depth 1 origin ${{SPACK_VERSION}} --no-tags && \
-    git checkout --quiet FETCH_HEAD
-ENV PATH="/root/spack/bin:${{PATH}}"
+ENV SPACK_VERSION=${{SPACK_VERSION:-2e7168b4cfe9222a94becf0c2c5b401f513181ec}}
+ARG SPACK_PACKAGES_VERSION
+ENV SPACK_PACKAGES_VERSION=${{SPACK_PACKAGES_VERSION:-ec6c0f3435d3a862337a7af71d07790f4d116274}}
```

Comment:
```suggestion
ENV SPACK_PACKAGES_VERSION=${{SPACK_PACKAGES_VERSION:-c63151a5baffcd96978966cd5ed83a6b8328e026}}
```
Forgot to update the Python script...